src/app/_components/MobileNavigation.tsx

```diff
@@ -73,23 +79,17 @@ type NestedMenuItemProps = {
 
 type NestedMenuProps = {
   title: string
-  items: NestedMenuItemProps[]
+  items: Array<NestedMenuItemProps>
```

Comment:
@CharlyMartin ~ We recently went in and changed all of this to `[]` - Barbara mentioned she prefers the simplicity of it - And I think I agree. Any reason for the `Array` use?

Reply:
Ah my bad, I didn't know.

I prefer `Array` now for the readability aspect. I read [this article](https://tkdodo.eu/blog/array-types-in-type-script) a while ago about it.

> We usually read from left to right, so the things that are more important should come first. We say: "this is an array of strings"

Happy with `[]` too!

Reply:
Thank you for sharing the article - Like the say - Let's implement at least https://typescript-eslint.io/rules/array-type/

Reply:
And let's create a ticket to talk about the article - It makes sense to me to move on to your notation.

Reply:
Happy to go with `Array`, my comment before was about using one format. Thanks for sharing @CharlyMartin!